src/lib/datepicker/datepicker.md

```diff
@@ -84,38 +101,54 @@ Each validation property has a different error that can be checked:
  * A value that violates the `matDatepickerFilter` property will have a `matDatepickerFilter` error.
  
 ### Input and change events
-The input's native `input` and `change` events will only trigger due to user interaction with the
-input element; they will not fire when the user selects a date from the calendar popup. Because of
-this limitation, the datepicker input also has support for `dateInput` and `dateChange` events.
-These trigger when the user interacts with either the input or the popup.
-  
-```html
-<input [matDatepicker]="d" (dateInput)="onInput($event)" (dateChange)="onChange($event)">
-<mat-datepicker #d></mat-datepicker>
-```
+
+The input's native `(input)` and `(change)` events will only trigger due to user interaction with
+the input element; they will not fire when the user selects a date from the calendar popup. Because
+of this limitation, the datepicker input also has support for `(dateInput)` and `(dateChange)`
+events. These trigger when the user interacts with either the input or the popup.
+
+The `(dateInput)` event will fire whenever the value changes due to the user typing or selecting a
+date from the calendar. The `(dateChange)` event will fire whenever the user finishes typing input
+(on `<input>` blur), or when the user chooses a date from the calendar.
+
+<!-- example(datepicker-events) -->
+
+### Disabling parts of the datepicker
+
+As with any standard `<input>`, it is possible to disable the datepicker input by adding the
+`disabled` property. By default, the `<mat-datepicker>` and `<mat-datepicker-toggle>` will inherit
+their disabled state from the `<input>`, but this can be overridden by setting the `disabled`
+property on the datepicker or toggle elements. This can be useful if you want to disable text input
+but allow selection via the calendar or vice-versa.
+
+<!-- example(datepicker-disabled) -->
 
 ### Touch UI mode
+
 The datepicker normally opens as a popup under the input. However this is not ideal for touch
 devices that don't have as much screen real estate and need bigger click targets. For this reason
-`mat-datepicker` has a `touchUi` property that can be set to `true` in order to enable a more touch
-friendly UI where the calendar opens in a large dialog.
+`<mat-datepicker>` has a `touchUi` property that can be set to `true` in order to enable a more
+touch friendly UI where the calendar opens in a large dialog.
 
 <!-- example(datepicker-touch) -->
 
 ### Manually opening and closing the calendar
+
 The calendar popup can be programmatically controlled using the `open` and `close` methods on the
-`mat-datepicker`. It also has an `opened` property that reflects the status of the popup.
+`<mat-datepicker>`. It also has an `opened` property that reflects the status of the popup.
 
 <!-- example(datepicker-api) -->
 
 ### Internationalization
+
 In order to support internationalization, the datepicker supports customization of the following
```

Comment:
While you're here, change this to
"Internationalization of the datepicker is configured via three aspects:"
?

Reply:
Also, it's odd that the headers directly under this list don't align with the points here.